jenkins/public_tests_all.Jenkinsfile

```diff
@@ -14,6 +14,11 @@ pipeline {
         sh "./build_openroad.sh --local";
         stash name: "install", includes: "tools/install/**";
       }
+      post {
+        success {
+          sh "mkdir failures"
+        }
+      }
```

Comment:
we run this on a distributed environment, is not guaranteed that the folder will be present on all machines. Move it to the failure stage below with the `-p` flag to avoid errors.